.devcontainer/Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN curl -L -o /usr/local/bin/pixi -fsSL --compressed "https://github.com/prefix
 # set some user and workdir settings to work nicely with vscode
 ENV VSCODE_HOME=/home/vscode \
     VSCODE_USER=vscode \
-    PIXI_ENV=${PIXI_ENV} \
+    CONDA_DEFAULT_ENV=${CONDA_DEFAULT_ENV} \
```

Comment:
Actually `CONDA_DEFAULT_ENV` is not the value of the pixi environment. So let's just remove this completely